docs/docs/quickstart/complete-guide.mdx

```diff
@@ -8,7 +8,7 @@ import TabItem from '@theme/TabItem';
 
 # Complete quickstart guide
 
-Sedge is a one-click setup tool which you can use to setup and run a validator on Ethereum easily and without any prior knowledge. In this guide you will go through every step in the validator setup using Sedge, along with detailed explanations for beginners. 
+Sedge is a one-click setup tool that you can use to set up and run a validator on Ethereum easily and without any prior knowledge. In this guide you will go through every step in the validator setup using Sedge, along with detailed explanations for beginners. 
 
 This is a beginner-friendly guide. Familiarity with the command line is expected, but otherwise this guide makes no assumptions about your technical skills or prior knowledge.
```

Comment:
```suggestion
This is a beginner-friendly guide. Familiarity with the command line is expected, but otherwise, this guide makes no assumptions about your technical skills or prior knowledge.
```